### PR TITLE
Add template dashboard page

### PR DESCRIPTION
**Pull request by vicancy**
A dashboard for #1134

### CODE REVIEW

**Comment by vicancy** [comment]
capture

File: Documentation/template/_footer.liquid [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{% comment %}Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.{% endcomment %}
+<footer>
+  <div class="grad-bottom"></div>
+  <div class="footer">
+    <div class="container">
+      <span class="pull-right">
+        <a href="#top">Back to top</a>
+      </span>
+    {% if _appFooter %}
```
Comment (hellosnow) [inline Documentation/template/_footer.liquid]: >{% [](start = 4, length = 2)

use "{%-“? to ignore this line spaces

**Comment by hellosnow** [comment]
can share with us the site in github.io?

---
In reply to: [289685152](https://github.com/dotnet/docfx/pull/1500#issuecomment-289685152) [](ancestors = 289685152)

File: Documentation/template/_footer.liquid [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    {{_appFooter}}
+    {% else %}
+    <span>Copyright © 2015-2016 Microsoft<br>Generated by <strong>DocFX</strong></span>
```
Comment (hellosnow) [inline Documentation/template/_footer.liquid]: >2016 [](start = 27, length = 4)

2017?

File: Documentation/template/dashboard.html.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+function transform(model, _attrs){
```
Comment (superyyrrzz) [inline Documentation/template/dashboard.html.js]: copyright header

**Comment by superyyrrzz** [comment]
:shipit:

File: Documentation/templates-and-plugins/contribute-your-template.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+﻿Add Your Template to the Dashboard
+----------------
+Add your own template info to the `templates` section of *YAML Header* of [templates-dashboard.md](https://github.com/dotnet/docfx/edit/dev/Documentation/templates-and-plugins/templates-dashboard.md), and it will be shown in the dashboard!
```
Comment (hellosnow) [inline Documentation/templates-and-plugins/contribute-your-template.md]: >https://github.com/dotnet/docfx/edit/dev/Documentation/templates-and-plugins/templates-dashboard.md [](start = 99, length = 99)

how about use relative path?

File: Documentation/template/_logo.liquid [hunk 5]
```diff
@@ -0,0 +1,4 @@
+{% comment %}Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.{% endcomment %}
+<a class="navbar-brand" href="{{_rel}}index.html">
+  <img id="logo" class="svg" src="{{_rel}}{{_appLogoPath}}{{^_appLogoPath}}logo.svg{{/_appLogoPath}}" alt="{{_appName}}" >
```
Comment (hellosnow) [inline Documentation/template/_logo.liquid]: >{{^ [](start = 58, length = 3)

is {{^...}} mustache syntax? liquid support it like {%- if x != null -%}?

File: Documentation/template/_navbar.liquid [hunk 6]
```diff
@@ -0,0 +1,21 @@
+{% comment %}Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.{% endcomment %}
+<nav id="autocollapse" class="navbar navbar-inverse ng-scope" role="navigation">
+  <div class="container">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
```
Comment (hellosnow) [inline Documentation/template/_navbar.liquid]: >span class="icon-bar"></span> [](start = 9, length = 29)

need we repeat by {%- for i in (1..3) -%}?

File: Documentation/template/_head.liquid [hunk 7]
```diff
@@ -0,0 +1,35 @@
+{% comment %}Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.{% endcomment %}
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  {% if title and _appTitle %}
+    <title>{{title}} | {{appTitle}}</title>
+    <meta name="title" content="{{title}} | {{appTitle}}">
+  {% else %}
+    {% if title or _appTitle %}
```
Comment (hellosnow) [inline Documentation/template/_head.liquid]: >if [](start = 7, length = 2)

minor: indent

File: Documentation/template/dashboard.html.js [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+      vm[key] = model[key];
+    }
+  }
+  // If toc is not defined in model, read it from _attrs
+  if (vm._tocPath && vm._tocPath.indexOf("~/") == 0) {
+    vm._tocPath = vm._tocPath.substring(2);
+  }
+  if (vm._navPath === vm._tocPath){
+    vm._allowToc = false;
+  }else{
```
Comment (hellosnow) [inline Documentation/template/dashboard.html.js]: >else [](start = 3, length = 4)

minor: format

File: Documentation/templates-and-plugins/contribute-your-template.md (hunk 4 above)
Comment (vicancy) [inline Documentation/templates-and-plugins/contribute-your-template.md]: it's *edit*

File: Documentation/template/_footer.liquid (hunk 1 above)
Comment (vicancy) [inline Documentation/template/_footer.liquid]: cool

File: Documentation/template/_navbar.liquid (hunk 6 above)
Comment (vicancy) [inline Documentation/template/_navbar.liquid]: don't think it is neccessory to use liquid syntax, as to keep the most similarity to mustache template, 

---
In reply to: [108393878](https://github.com/dotnet/docfx/pull/1500#discussion_r108393878) [](ancestors = 108393878)

**Comment by vicancy** [comment]
sample page: https://vicancy.github.io/docfx/templates-and-plugins/templates-dashboard.html

**Comment by hellosnow** [comment]
looks good :)

---
In reply to: [289994450](https://github.com/dotnet/docfx/pull/1500#issuecomment-289994450) [](ancestors = 289994450)

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**